### PR TITLE
Implement updatePlan path in the RecurlyClient

### PR DESCRIPTION
**Pull request by bhelx**
Addresses Issue https://github.com/killbilling/recurly-java-library/issues/105 reported by @shrihubble 

See the integration test for details

### CODE REVIEW

File: src/main/java/com/ning/billing/recurly/RecurlyClient.java [hunk 1]
```diff
@@ -619,6 +619,17 @@ public Plan createPlan(final Plan plan) {
     }
 
     /**
+     * Update a Plan's info
+     * <p>
```
Comment (pierre) [inline src/main/java/com/ning/billing/recurly/RecurlyClient.java]: :+1:

FYI, `<p/>` won't work anymore out of the box on JDK 1.8.

**Comment by pierre** [comment]
:tada:

**Comment by shrihubble** [comment]
@bhelx 

Can you confirm whether 0.5.0 version has updatePlan method in it? When I download the JAR from http://mvnrepository.com/artifact/com.ning.billing/recurly-java-library/0.5.0 . I don's see updatePlan method inside it. Please let me know.

Thank U.

**Comment by bhelx** [comment]
@shrihubble I don't think this made it into the 0.5.0 release. @pierre would you consider doing a 0.5.1 release or could you recommend a way for @shrihubble to point to master?

**Comment by pierre** [comment]
@shrihubble This isn't in 0.5.0. For now, you need to build the library locally.

@bhelx I was actually waiting for #96 before doing another release. Let me know if you think it's going to take some time though (no rush).

**Comment by bhelx** [comment]
@pierre I'll take a stab at it today

**Pull request merged**